src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -2,17 +2,23 @@
 # FORMAT:
 # 'Category': Comma separate list of 'StartId-EndId' or 'Id' or 'Prefix'
 
-# FxCop rules
-Design: CA1000-CA1099, CA2210
-Globalization: CA1300-CA1399, CA2101
-Mobility: CA1600-CA1699
-Performance: CA1800-CA1899, HA
-Security: CA2100-CA2199, CA2300-CA2399, CA3000-CA3147, CA5300-CA5499
-Usage: CA1801, CA1806, CA1816, CA2200-CA2209, CA2211-CA2299
-Naming: CA1700-CA1799
-Maintainability: CA1500-CA1599
-Reliability: CA2000-CA2099, CA9999
-Documentation: CA1200-CA1299
+# Code Analysis (CA) rules - https://docs.microsoft.com/visualstudio/code-quality/code-analysis-for-managed-code-warnings
+#
+# The ranges below represent the **currently used IDs** for the corresponding category.
+# When implementing a new rule for the category:
+#   1. Choose the rule ID immediately following the range end.
+#   2. Update the range end to the chosen rule ID.
+#
+Design: CA2210, CA1000-CA1069
+Globalization: CA2101, CA1300-CA1309
+Mobility: CA1600-CA1601
+Performance: HA, CA1800-CA1829
+Security: CA2100-CA2153, CA2300-CA2399, CA3000-CA3147, CA5300-CA5499
```

Comment:
@dotpaul Let me know if you'd like to make a similar change to switch from reserved IDs to currently used IDs. I can create a separate PR on 2.9.x branch just for security rules.

Reply:
Sure, that'd be great

Reply:
@dotpaul I have updated the ranges and retargeted this PR to 2.9.x branch.